backend/climateconnect_api/views/user_views.py

```diff
@@ -151,6 +152,10 @@ class ListMemberProfilesView(ListAPIView):
     search_fields = ['name']
     serializer_class = UserProfileStubSerializer
 
+    @method_decorator(cache_page(settings.DEFAULT_CACHE_TIMEOUT))
```

Comment:
Could we get away with just:

```suggestion
    @cache_page(settings.DEFAULT_CACHE_TIMEOUT)
```